tests/test_events.py

```diff
@@ -50,23 +52,64 @@ def setUp(self):
 
     def test_record_event_for_bundle(self):
         metadata_document = dict(foo=f"{uuid4()}")
-        key = f"bundles/{uuid4()}.{datetime_to_version_format(datetime.utcnow())}"
-        test_parameters = [(replica, pfxs) for replica in Replica for pfxs in [None, ("foo", "bar")]]
-        for replica, prefixes in test_parameters:
-            with self.subTest(replica=replica.name, flashflood_prefixes=prefixes):
-                self._test_record_event_for_bundle(replica, prefixes, metadata_document, key)
-
-    def _test_record_event_for_bundle(self, replica, prefixes, metadata_document, key):
-        with mock.patch("dss.events._build_bundle_metadata_document", return_value=metadata_document):
+        keys = [f"bundles/{uuid4()}.{datetime_to_version_format(datetime.utcnow())}",
+                f"bundles/{uuid4()}.{datetime_to_version_format(datetime.utcnow())}.dead",
+                f"bundles/{uuid4()}.dead"]
+        test_parameters = [(replica, key, pfxs)
+                           for replica in Replica
+                           for key in keys
+                           for pfxs in [None, ("foo", "bar")]]
+        for replica, key, prefixes in test_parameters:
+            with self.subTest(replica=replica.name, key=key, flashflood_prefixes=prefixes):
```

Comment:
Huh, didn't know you could pass kwargs only to a subTest name. How does this subTest name look when displayed on the screen?